KatyaBebeshko/HW6/task2.py

```diff
@@ -0,0 +1,18 @@
+# Task2. Write a script that checks the login that the user enters.
+# If the login is "First", then greet the users. If the login is different, send an error message.
+# (need to use loop while)
+
```

Comment:
good job
but the task condition did not predict the number of trials

Reply:
agree

here is a correct way

while input('Enter login: ') != 'First':
    print('Access denied. Try again.')
else:
    print('Successful login')